OutSystems/outsystems-maps
# Update Images.css

This PR Includes the changes made in ROU-4361 in order to replace the base 64 for the CSS vars → `--map-container-image-google` and `--map-container-image-leaflet`

### What was happening
* Based on feedback from the Architecture group, we’re having some issues with modules while creating new apps, mainly when referencing OutSystems Maps.
* The biggest impact is related to the Google Maps and Leaflet preview images
*  Module currently has 6721KB
* CSS var `--map-container-image-google` = 252.7 KB, 258763 chars
* CSS var `--map-container-image-leaflet` = 1.96 MB, 2056778 chars

### What was done
* Changed:
  * **--map-container-image-google:**
           * Current Base64 = Size : 252.7 KB, 258763 chars
           * New Base64 = Size : 171.28 KB, 175404 chars
           * Image = 190KB
           * New Image = 129KB
           
  * **--map-container-image-leaflet:**
    * Current Base64 = Size : 1.96 MB, 2056778 chars
    * New Base64 = 561.29 KB, 574779 chars
    * Image = 1507KB
    * New Image = 421KB
            
  * **Module:**
    * Current Module = 6721KB
    * After Changes = 3250KB 

### Test Steps
1. Go to the module `Maps_Automation` in Service Studio
2. Open a screen using Maps from Google Maps
3. Check the preview and manipulate the input parameters that have an impact on the design time visualization
4. Do the same tests on a screen using Leaflet


### Checklist
